presto-main/src/main/java/com/facebook/presto/testing/MaterializedRow.java

```diff
@@ -54,7 +54,7 @@ private static Object processValue(int precision, Object value)
     {
```

Comment:
Commit message is not very clear and too contextual. Take a look at my commit message for integer.